File: backend/micro_epoweredmove_poi_chargingstation/src/main/java/com/kosdiam/epoweredmove/models/enums/PaymentType.java
package com.kosdiam.epoweredmove.models.enums;

public enum PaymentType {
    APP("App"),
    PAYPAL("Paypal"),
    EBANKING("E-banking"),
    LOCALLY("Locally");

    private String paymentType;
    private PaymentType(String paymentType){
        this.paymentType = paymentType;
    }

    @Override
    public String toString(){
        return paymentType;
    }
}

File: frontend/E-PoweredMoveAndroidApp/app/src/main/java/com/kosdiam/epoweredmove/models/enums/PaymentType.kt
package com.kosdiam.epoweredmove.models.enums

import com.kosdiam.epoweredmove.R
import com.kosdiam.epoweredmove.models.interfaces.IEnumName

enum class PaymentType (override val stringResource: Int) : IEnumName {
    APP(R.string.payment_type_app),
    PAYPAL(R.string.payment_type_paypal),
    EBANKING(R.string.payment_type_ebanking),
    LOCALLY(R.string.payment_type_locally);
}